src/status_im/translations/en.cljs

```diff
@@ -472,11 +472,11 @@
    :wallet-settings                      "Wallet settings"
    :wallet-manage-assets                 "Manage Assets"
    :sign-message                         "Sign Message"
-   :signing-message-phrase-description   "Sign the message by entering your password. Make sure that the words above match your secret signing phrase"
-   :signing-phrase-description           "Sign the transaction by entering your password. Make sure that the words above match your secret signing phrase"
+   :signing-message-phrase-description   "If you recognize these words, enter your login password to sign the message"
+   :signing-phrase-description           "If you recognize these words, enter your login password to sign the transaction"
```

Comment:
"login password" sounds strange to me. We have no other password in the app, so I suggest to remove word "login"

Reply:
In testing we've observed that users get confused about what to enter here. They think they need to enter the three words or sometimes even the seed phrase. I'm attempting to make it more obvious this it's the password they use to open the app that's needed.